domain/build.gradle

```diff
@@ -49,7 +49,9 @@ dependencies {
       'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03',
       'com.google.dagger:dagger:2.24',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
-      'androidx.exifinterface:exifinterface:1.0.0-rc01'
+      'androidx.exifinterface:exifinterface:1.0.0-rc01',
```

Comment:
Nit: We are actually following alphabetical order here.

Reply:
Done.